carta/util.py

```diff
@@ -88,7 +88,14 @@ def __repr__(self):
     def json(self):
         """The JSON serialization of this object."""
         return {"macroTarget": self.target, "macroVariable": self.variable}
+    
+class Undefined(Macro):
+    """
+    A subclass of Macro to construct a placeholder for `"undefined"`.
+    """
 
+    def __init__(self, target="", variable="undefined"):
```

Comment:
This constructor shouldn't take any parameters. You should just pass these literals into the parent constructor below.